cargo-dist/src/backend/installer/mod.rs

```diff
@@ -66,8 +66,8 @@ pub struct InstallerInfo {
 pub struct ExecutableZipFragment {
     /// The id of the artifact
     pub id: String,
-    /// The targets the artifact supports
-    pub target_triples: Vec<TargetTriple>,
+    /// The target the artifact supports
+    pub target_triples: TargetTriple,
```

Comment:
The fact that everything ignored that this can be an array sucked, it's nice that the new impl smooths this out. The value should be renamed but my rust-analyzer was bugging out so I opted not to do this.